zazu/plugins/eslint_styler.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""ESLint plugin for zazu."""
+import zazu.styler
+zazu.util.lazy_import(locals(), [
+    'subprocess',
+    'os',
+    'tempfile'
+])
+
+__author__ = "Patrick Moore"
+__copyright__ = "Copyright 2018"
+
+
+class ESLintStyler(zazu.styler.Styler):
+    """ESLint plugin for code styling."""
+
+    def style_string(self, string):
+        """Fix a string to be within style guidelines."""
+        temp = tempfile.NamedTemporaryFile(delete=False, suffix=".js")
+        temp_path = temp.name
+        args = ['eslint', '--fix'] + self.options + [temp_path]
+        temp.write(string)
+        temp.close()
+        try:
+            subprocess.check_output(args)
+        except subprocess.CalledProcessError:
+            pass
```

Comment:
This is a hard one. This will basically swallow ANY non zero return from ESLint. If ESlint fails for some reason like a invalid syntax error or more likely if someone passes invalid flags to it we don't want to ignore that. I'm not sure how to detect that however. It seemed like ESLint returns a non zero return code if style changes were made it may be difficult to detect an actual error without examining stderr.  Is it possible to use the `--quiet` or `--max-warnings=2147483648` options to get a 0 return code in the things worked fine but changes were made case?